tracex_project/tracex/logic/utils.py

```diff
@@ -191,24 +191,45 @@ class DataFrameUtilities:
     def get_events_df(query: Q = None):
         """Get all events from the database, or filter them by a query and return them as a dataframe."""
         traces = Trace.manager.all() if query is None else Trace.manager.filter(query)
+
         if not traces.exists():
             return pd.DataFrame()  # Return an empty dataframe if no traces are found
 
         event_data = []
+
         for trace in traces:
             events = trace.events.all()
             for event in events:
-                event_data.append(
-                    {
-                        "case:concept:name": trace.id,
-                        "activity": event.activity,
-                        "event_type": event.event_type,
-                        "time:timestamp": event.start,
-                        "time:end_timestamp": event.end,
-                        "time:duration": event.duration,
-                        "attribute_location": event.location,
-                    }
-                )
+                event_dict = {
```

Comment:
the column names are wrong, resulting in key errors when executing the pipeline. please rename them to what they were before.